scss/_layouts_fluid-breakout.scss

```diff
@@ -0,0 +1,199 @@
+
+    @media (min-width: $l-fluid-breakout-3-col-threshold) {
+      // display: flex;
+      flex: 1 1 auto;
+      width: $l-fluid-breakout-aside-width;
+    }
+
+    @media (min-width: $threshold-4-6-col) and (max-width: $l-fluid-breakout-3-col-threshold) {
+      padding-right: map-get($grid-margin-widths, large);
+    }
+  }
+
+  .l-fluid-breakout#{$suffix} {
+    @extend %l-fluid-breakout-ie11-fallback#{$suffix};
+
+    @supports (display: grid) {
+      grid-gap: 0 0;
+    }
+
+    @media (min-width: $breakpoint-large) {
+      margin-left: auto;
+      margin-right: auto;
+      max-width: 2 * $l-fluid-breakout-aside-width + $l-fluid-breakout-max-width;
+
+      @supports (display: grid) {
+        display: grid;
+        grid-template-columns:
+          minmax($l-fluid-breakout-aside-width, 1fr)
+          minmax(0, $l-fluid-breakout-max-width)
+          minmax($l-fluid-breakout-aside-width, 1fr);
+        grid-template-rows: auto;
+      }
+    }
+
+    @media (max-width: $threshold-4-6-col) {
+      padding-left: map-get($grid-margin-widths, small);
+      padding-right: map-get($grid-margin-widths, small);
+    }
+
+    @media (min-width: $threshold-4-6-col) and (max-width: $threshold-6-12-col) {
+      padding-left: map-get($grid-margin-widths, medium);
+      padding-right: map-get($grid-margin-widths, medium);
+    }
+
+    @media (min-width: $threshold-6-12-col) and (max-width: $l-fluid-breakout-3-col-threshold) {
+      padding-left: map-get($grid-margin-widths, large);
+      padding-right: map-get($grid-margin-widths, large);
+    }
+
+    @media (min-width: $l-fluid-breakout-3-col-threshold) {
+      padding-left: 0;
+      padding-right: 0;
+
+      @supports (display: grid) {
+        grid-gap: 0 0;
+      }
+    }
+
+    %l-fluid-breakout__main-and-full-common {
+      @extend %l-fluid-breakout__full-and-main-ie11-fallback#{$suffix};
+
+      width: 100%;
+
+      @supports (display: grid) {
+        display: grid;
+        grid-gap: 0 $grid-gap;
+        grid-row: 1;
+        grid-template-columns: repeat(auto-fit, minmax($l-fluid-breakout-main-child-width, 1fr));
+      }
+
+      @media (min-width: $l-fluid-breakout-3-col-threshold) {
+        padding-left: map-get($grid-margin-widths, large);
+        padding-right: map-get($grid-margin-widths, large);
+      }
+
+      & > * {
+        @supports (display: grid) {
+          // overwrite fallback margins
+          margin-left: 0;
+          margin-right: 0;
+        }
+      }
+    }
+
+    .l-fluid-breakout__full {
+      @extend %l-fluid-breakout__main-and-full-common;
+
+      @media (min-width: $breakpoint-large) {
+        // stylelint-disable-next-line max-nesting-depth
+        @supports (display: grid) {
+          grid-column: 1 / span 3;
+        }
+      }
+    }
+
+    .l-fluid-breakout__main {
+      @extend %l-fluid-breakout__main-and-full-common;
+
+      @media (min-width: $breakpoint-large) {
+        @supports (display: grid) {
```

Comment:
Would it make sense to wrap everything (that is not IE fallback) into `@supports (display: grid)`?
Right not it seems you separately use `@supports` in every single selector/query.

Reply:
some of the styles - like padding - shouldn't be wrapped in supports, that's why it is wrapped in smaller pieces